modules/ballerina-builtin/src/main/ballerina/ballerina/builtin/stringlib.bal

```diff
@@ -5,3 +5,98 @@ package ballerina.builtin;
 @Param { value:"anotherString: The string to be compared" }
 @Return { value:"boolean: True if the strings are equal; false otherwise" }
 public native function <string mainString> equalsIgnoreCase (string anotherString) (boolean);
+
+@Description { value:"Returns a string with all the characters converted to uppercase"}
+@Param { value:"s: The original string argument" }
+@Return { value:"string: The string converted to uppercase" }
+public native function <string s> toUpperCase () (string);
```

Comment:
Shall we either use <string mainString> or <string s> throughout the whole file to maintain the consistency.